mix.exs

```diff
@@ -21,8 +21,10 @@ defmodule Politex.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      # {:dep_from_hexpm, "~> 0.3.0"},
-      # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"},
+      {:httpoison, "~> 1.3.0"},
+      {:poison, "~> 4.0.1"},
```

Comment:
The community is largely moving to [jason](https://github.com/michalmuskala/jason).  It's faster than poison, and many projects have jumped ship already.  I'd recommend using it from the start.

Reply:
👍